### PR TITLE
A ResizeObserver HD-DPI example

### PR DESCRIPTION
**Pull request by greggman**
#356

I'm not sure if adding image-rendering fits here. Maybe that should be a separate example?

### CODE REVIEW

**Comment by kainino0x** [comment]
> I'm not sure if adding image-rendering fits here. Maybe that should be a separate example?

Probably. The only reason to put it here would be if we want to demo image-rendering at exact scales (like 1/2x, 1/4x of the native display resolution), but I don't think that's needed.

**Review by kainino0x — APPROVED** [review]
LGTM

File: sample/resizeObserverHDDPI/main.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      binding: 0,
+      resource: { buffer: uniformBuffer },
+    },
+  ],
+});
+
+const settings = {
+  color0: '#FF0000',
+  color1: '#00FFFF',
+  size: 1,
```
[the remark this answers is not in the document]
Comment (greggman) [reply sample/resizeObserverHDDPI/main.ts:61]: set to 4

File: sample/resizeObserverHDDPI/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      }
+      .resizable {
+        resize: both;
+        overflow: scroll;  /* required because resize doesn't work with overflow: visible, the default */
+        max-width: 100%;
+        max-height: 100%;
+      }
+    </style>
+    <script defer src="main.js" type="module"></script>
+    <gscript defer type="module" src="../../js/iframe-helper.js"></script>
```
[the remark this answers is not in the document]
Comment (greggman) [reply sample/resizeObserverHDDPI/index.html:29]: thanks for catching. I meant to delete that line for this sample.

**Pull request merged**